### DISCUSSION TITLE
[Feature Request] Support for multiple views

### DISCUSSION THREAD

**Discussion by dominicpratt** [LeoKlaus/Paperparrot#82] [Implemented]
Hey,

it would be nice if Paperparrot would support different views, like Paperless-NGX does:

![Bildschirm­foto 2023-02-22 um 11 25 13](https://user-images.githubusercontent.com/14056131/220593108-3fae80f4-b587-46c0-afb3-d9c0cf2f7f2e.png)
![Bildschirm­foto 2023-02-22 um 11 25 17](https://user-images.githubusercontent.com/14056131/220593114-ff7330a7-8b16-491d-b079-f46bc96b6623.png)
[Replaced original image as it contained possibly personal information]
![image](https://user-images.githubusercontent.com/66492595/220595192-13c5658e-2360-45ec-82bc-c402752948fc.png)

Especially the List View would be much clearer on the iPad/iPhone than the current view - I guess.

**Comment by LeoKlaus** [comment]
I like that idea, moving it to discussions. Thank you for your feedback!

**Comment by LeoKlaus** [comment]
A first version of this was implemented in 1.1.0
